c2corg_ui/templates/image/view.html

```diff
@@ -0,0 +1,161 @@
+<%block name="metarobots">
+  % if version:
+    <meta name="robots" content="noindex,follow">
+  % else:
+    <meta name="robots" content="index,follow">
+  % endif
+</%block>
+
+<%block name="moduleConstantsValues">
+  module.value('mapFeatureCollection', {
+    "type": "FeatureCollection",
+    "properties": {},
+    "features": [{
+      "type": "Feature",
+      "geometry": {"type": "Point", "coordinates": [901624.4172679152, 5858812.695644485]},
+      "properties": {
+        "title": ${dumps(locale['title']) | n},
+        "lang": "${locale['lang']}",
+        "documentId": ${image['document_id']},
+        "module": "images"
+      }
+    }]
+  });
+
+  module.value('documentData', {
+    "document_id": ${image.get('document_id')},
+    "lang": "${lang}",
+    "type": "i"
+    % if not version:
+       , "associations": ${dumps(image.get('associations')) | n}
+    % endif
+  });
+</%block>
+
+<header class="view-details-title">
+  <h1 class="routes">
+    <div class="icons-header">${get_route_activities(image, 'top')}</div>
+    <span class="title">${show_route_title(locale)}</span><br>
+    <label class="badge" onClick="window.location='${request.route_url('images_index')}' ">
+      <span class="glyphicon glyphicon-arrow-right"></span>&nbsp;<span translate>image</span>
+    </label>
+  </h1>
+</header>
+
+% if image.get('areas'):
+  <article class="location-static">
+    <span class="pin"></span>
+    ${show_areas(image)}
+  </article>
+% endif
+
+<section class="view-details-section images" app-view-details>
+
+  % if version:
+    ${show_archive_data('images', image, locale, version)}
+  % endif
+
+  <div class="view-details-informations col-xs-12  informations">
+    <h3 class="heading informations" ng-click="mainCtrl.animateHeaderIcon($event)" data-toggle="collapse" data-target="#document-informations">
+      <span translate>Informations</span><span class="glyphicon glyphicon-menu-down"></span>
+    </h3>
+    <section id="document-informations" class="collapse in">
+      ${get_image_general(image)}
+      ${get_image_camera_settings(image)}
+    </section>
+  </div>
+
+  ## DESCRIPTION
+  % if locale['summary'] or locale['description']:
+  <div class="view-details-description col-xs-12  description">
+    <h3 class="heading show-phone"><span translate>personal comments</span></h3>
+    <span class="lead">
+      <div id="document-description" class="collapse in">
+
+        % if locale['summary']:
+        <summary class="document-summary">
+          <label translate>summary</label><br>
+          ${show_attr(locale, 'summary')}
+        </summary>
+        % endif
+
+        % if locale['description'] :
+        ${show_attr(locale, 'description')}
+        % endif
+
+      </div>
+    </span>
+  </div>
+  % endif
+
+  <div class="view-details-description col-xs-12  description">
+    <span class="lead">
+      <img class="image" src="${image_url}/${image['filename']}" alt="${locale['title']}">
```

Comment:
Do you use the full size file here?
If yes, it's probably better to use a resized version with a link towards the full size file. See for instance in v5: http://www.camptocamp.org/images/796074/fr/bietschhorn